src/routes/auth.tsx

```diff
@@ -0,0 +1,60 @@
+import {
+  Box,
+  Button,
+  Container,
+  CssBaseline,
+  TextField,
+  ThemeProvider,
+  Typography,
+} from "@mui/material";
+import { darkTheme } from "../theme";
+
+export default function Auth() {
+  return (
+    <ThemeProvider theme={darkTheme}>
+      <Container component="main" maxWidth="xs" disableGutters>
+        <CssBaseline />
+        <Box
+          sx={{
+            marginTop: 8,
+            display: "flex",
+            flexDirection: "column",
+            alignItems: "center",
+          }}
+        >
```

Comment:
Use <Stack>, it will be much cleaner and you will learn how to use it. As you are going to use it a LOT. have to get used to how it works.